hackathon2.R

```diff
@@ -0,0 +1,90 @@
+# GOAL of this research: ARE WE IMPROVING on SCREENING of MUAC and OEDEMA? 
+##Are wee expanding our screening coverage? From baseline vs. endline? By sex? By location?
+
+## Install & Load R packages 
+install.packages("gtsummary")
+install.packages("dplyr")
```

Comment:
You should avoid adding `install.packages()` code into an RScript or workflow especially one that is already using `renv` package to manage dependencies.

RScripts/R workflows should only have calls to load the libraries as you have it in lines 8 and 9.